registry/kubevirt/0.14.0/kubevirtoperator.v0.14.0.clusterserviceversion.yaml

```diff
@@ -52,6 +52,11 @@ spec:
     spec:
       permissions:
       - serviceAccountName: kubevirt-operator
+        metadata:
```

Comment:
Does OLM allow you to add this metadata?

Reply:
Good question.
It's part of the original manifest so I guess that this is required, so if OLM doesn't allow or simply disregards this, then we may have an issue here.

Reply:
Yup. I don't think we can do this https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Documentation/design/building-your-csv.md#operator-install

We'll need to do something else.